test/parallel/test-fs-stat-bigint.js

```diff
@@ -122,6 +122,23 @@ if (!common.isWindows) {
   fs.closeSync(fd);
 }
 
+const runSyncFailureTest = (func, arg, error) => {
+  assert.throws(() => func(arg), error);
+  assert.strictEqual(func(arg, { throws: false }), undefined);
+};
+
+{
+  runSyncFailureTest(fs.statSync, 'does_not_exist', { code: 'ENOENT' });
+}
+
+{
+  runSyncFailureTest(fs.lstatSync, 'does_not_exist', { code: 'ENOENT' });
+}
+
+{
+  runSyncFailureTest(fs.fstatSync, 9999, { code: 'EBADF' });
```

Comment:
Nit: I'd just inline these to make the clearer/simpler.

Reply:
Personally, I find this way more readable.  However, I'll switch it if anyone else supports the change.